src/styles/tabs.js

```diff
@@ -4,9 +4,10 @@ import { BRAND_COLOR } from './';
 type Props = {
   showLabel: boolean,
   showIcon: boolean,
+  style?: Object,
 };
 
-export default ({ showLabel, showIcon }: Props) => ({
+export default ({ showLabel, showIcon, style = {} }: Props) => ({
```

Comment:
`style` does not need to have a default value.

Reply:
Done 👍